.github/workflows/Tests.yaml

```diff
@@ -438,3 +438,59 @@ jobs:
           # We get occasional random timeouts, repeat tests to see if
           # it is a random timeout or systematic
           ctest -j2 --output-on-failure -L inputfiles --repeat after-timeout:3
+      - name: Test formaline tar can be built
+        working-directory: /work/build
+        run: >
+          make EvolveBurgersStep -j2
+
+          if [ ! -f ./bin/EvolveBurgersStep ]; then
+            echo "Could not find the executable EvolveBurgersStep";
+            echo "which we use for testing formaline";
+            exit 1
+          fi
+
+          ASAN_OPTIONS=detect_leaks=0 ./bin/EvolveBurgersStep
```

Comment:
why do you run this with ASAN?

Reply:
I disable ASAN memory leak checking (false positives with Charm++). We have a build that runs with ASAN because we want to avoid memory issues and so I need to disable the leak checking here. CMake does this for all our input file tests internally